src/braket/aws/aws_quantum_job.py

```diff
@@ -165,6 +165,11 @@ def create(
         Raises:
             ValueError: Raises ValueError if the parameters are not valid.
         """
+
+        if(type(hyperparameters) is dict):
```

Comment:
```suggestion
        if isinstance(hyperparameters, dict):
```

Reply:
Actually, we shouldn't even be checking that hyperparameters is a dict here. The documentation and annotation clearly states that we expect a dict, with a default value of `{}`. This check should be removed and this functionality should be moved to `prepare_quantum_job`, specifically after the default value is propagated [here](https://github.com/aws/amazon-braket-sdk-python/blob/5eb611d17fa521c545833142af522cf8eef4e5c1/src/braket/jobs/quantum_job_creation.py#L144)

Reply:
Sounds good, I moved it to `prepare_quantum_job `.